Unable to install the plugin

There is a problem with plugin dependency "com.verso.cordova.clipboard":

`$ cordova plugin add --save cordova-instagram-plugin
Installing "cordova-instagram-plugin" for android
Failed to install 'cordova-instagram-plugin': Error
    at /home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-lib/src/plugman/fetch.js:205:33
    at _rejected (/home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:864:24)
    at /home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:890:30
    at Promise.when (/home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:1142:31)
    at Promise.promise.promiseDispatch (/home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:808:41)
    at /home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:624:44
    at runSingle (/home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:137:13)
    at flush (/home/mohammad/.nvm/versions/node/v7.4.0/lib/node_modules/cordova/node_modules/cordova-fetch/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
Error: Failed to fetch plugin https://github.com/VersoSolutions/CordovaClipboard.git via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 235 Error output:
npm ERR! addLocal Could not install /tmp/npm-29219-f531f099/git-cache-fda9d397/03fe48b62411cbff22229ca13cc3ac8b282f7945
npm ERR! Linux 4.4.0-75-generic
npm ERR! argv "/home/mohammad/.nvm/versions/node/v7.4.0/bin/node" "/home/mohammad/.nvm/versions/node/v7.4.0/bin/npm" "install" "https://github.com/VersoSolutions/CordovaClipboard.git" "--save"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! code EISDIR
npm ERR! errno -21
npm ERR! syscall read

npm ERR! eisdir EISDIR: illegal operation on a directory, read
npm ERR! eisdir This is most likely not a problem with npm itself
npm ERR! eisdir and is related to npm not being able to find a package.json in
npm ERR! eisdir a package you are trying to install.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/mohammad/git/jetcamp-app/node_modules/npm-debug.log
`

--- a/plugin.xml
+++ b/plugin.xml
@@ -11,8 +11,8 @@
   <license>MIT</license>
 
   <dependency
-    id="com.verso.cordova.clipboard"
-    url="https://github.com/VersoSolutions/CordovaClipboard.git">
+    id="cordova-universal-clipboard"
+    url="https://github.com/Visigo/CordovaClipboard.git">
   </dependency>
 
   <js-module src="www/CDVInstagramPlugin.js" name="InstagramPlugin">

